{{cookiecutter.github_repository}}/.editorconfig

```diff
@@ -15,7 +15,7 @@ indent_size = 4
 [*.py]
 # https://github.com/timothycrosley/isort/wiki/isort-Settings
 # https://timothycrosley.github.io/isort/docs/configuration/profiles/#black
-line_length=88
+line_length=120
 known_first_party={{ cookiecutter.main_module }}
```

Comment:
If we are using `pyproject.toml` for keeping isort and black config, I suggest we remove these. Otherwise it will be confusing in the future.